ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -426,7 +426,7 @@ private boolean isHardwareKeyboardAvailable() {
     return getContext().getResources().getConfiguration().keyboard != Configuration.KEYBOARD_NOKEYS;
   }
 
-  protected void showSoftKeyboard() {
+  protected boolean showSoftKeyboard() {
```

Comment:
There is a bug in this area, any chance this could be related?

https://office.visualstudio.com/OC/_workitems/edit/3522872

Reply:
No, this is a very recent and small change that does not go back as far as that bug. I've changed this method's signature on FB's side to be protected, as we need it in RichEdit, but I am bringing back the boolean to match FB, as it seemed to be an unnecessary change on our side.